### PR TITLE
Ft feedback emojis

### PR DESCRIPTION
**Pull request by josphatmwania**
# Scope

_Please make sure to read the [Contribution Guidelines](https://github.com/droidconKE/droidconKE2023Android/blob/main/CONTRIBUTING.md)
and check that you understand and have followed it as best as possible Explain what your feature
does in a short paragraph._ please check the below boxes
- [x] I have followed the coding conventions
- [ ] I have added/updated necessary tests
- [x] I have tested the changes added on a physical device
- [x] I have run `./codeAnalysis.sh` on linux/unix or `codeAnalysys.bat` on windows to make sure all lint/formatting checks have been done.

## Closes/Fixes Issues
_Declare any issues by typing `fixes #1` or `closes #1` for example so that the automation can kick
in when this is merged_

## Other testing QA Notes
_What have you tested specifically and what possible impacts/areas there are that may need retesting
by others._

Please add a screenshot (if necessary)
image

### CODE REVIEW

File: presentation/src/main/java/com/android254/presentation/feedback/view/FeedBackScreen.kt [hunk 1]
```diff
@@ -167,10 +167,10 @@ fun FeedBackScreen(
                                     modifier = Modifier
                                         .height(40.dp)
                                         .width(40.dp),
-                                    painter = painterResource(id = R.drawable.smiling),
+                                    painter = painterResource(id = R.drawable.ic_feedback_bad_face),
                                     contentDescription = stringResource(id = R.string.sign_in_label)
                                 )
-                                Text(text = "Okay")
+                                Text(text = "Bad")
```
Comment (chepsi) [inline presentation/src/main/java/com/android254/presentation/feedback/view/FeedBackScreen.kt:173]: can we extract this to string resource?

File: presentation/src/main/java/com/android254/presentation/feedback/view/FeedBackScreen.kt [hunk 2]
```diff
@@ -183,10 +183,10 @@ fun FeedBackScreen(
                                     modifier = Modifier
                                         .height(40.dp)
                                         .width(40.dp),
-                                    painter = painterResource(id = R.drawable.smiling),
+                                    painter = painterResource(id = R.drawable.ic_feedback_neutral_face),
                                     contentDescription = stringResource(id = R.string.sign_in_label)
                                 )
-                                Text(text = "Great")
+                                Text(text = "Okay")
```
Comment (chepsi) [inline presentation/src/main/java/com/android254/presentation/feedback/view/FeedBackScreen.kt:189]: can we extract this to string resource?

File: presentation/src/main/java/com/android254/presentation/feedback/view/FeedBackScreen.kt [hunk 3]
```diff
@@ -199,10 +199,10 @@ fun FeedBackScreen(
                                     modifier = Modifier
                                         .height(40.dp)
                                         .width(40.dp),
-                                    painter = painterResource(id = R.drawable.smiling),
+                                    painter = painterResource(id = R.drawable.ic_feedback_smiling_face),
                                     contentDescription = stringResource(id = R.string.sign_in_label)
                                 )
-                                Text(text = "Bad")
+                                Text(text = "Great")
```
Comment (chepsi) [inline presentation/src/main/java/com/android254/presentation/feedback/view/FeedBackScreen.kt:205]: can we extract this to string resource?

**Review by chepsi — APPROVED** [review]

File: presentation/src/main/java/com/android254/presentation/feedback/view/FeedBackScreen.kt (hunk 3 above)
Comment (josphatmwania) [reply presentation/src/main/java/com/android254/presentation/feedback/view/FeedBackScreen.kt:205]: Yes. I'll fix that.

Comment (josphatmwania) [reply presentation/src/main/java/com/android254/presentation/feedback/view/FeedBackScreen.kt]: @chepsi Done. Have a look a the changes.

**Review by chepsi — APPROVED** [review]

**Pull request merged**